From 3d1fe557be777874b234fcf2b456947189bdc24e Mon Sep 17 00:00:00 2001
From: Baris Dinc <barisdinc@gmail.com>
Date: Sun, 11 Oct 2026 13:59:06 +0300
Subject: [PATCH 2/4] configs: zynq_pluto: Rebrand as yATOLYE DATV Repeater,
 add ffmpeg and lame

Set the hostname to atolye-datv-repeater and the login banner to
"Welcome to yATOLYE DATV Repeater". Add ffmpeg and lame to the root
filesystem for on-device audio/video processing.
---
 configs/zynq_pluto_defconfig | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/configs/zynq_pluto_defconfig b/configs/zynq_pluto_defconfig
index 6203766e04..541c2275b5 100644
--- a/configs/zynq_pluto_defconfig
+++ b/configs/zynq_pluto_defconfig
@@ -6,8 +6,8 @@ BR2_ARM_FPU_NEON=y
 BR2_TOOLCHAIN_EXTERNAL=y
 BR2_TOOLCHAIN_EXTERNAL_LINARO_ARM=y
 BR2_OPTIMIZE_2=y
-BR2_TARGET_GENERIC_HOSTNAME="pluto"
-BR2_TARGET_GENERIC_ISSUE="Welcome to Pluto"
+BR2_TARGET_GENERIC_HOSTNAME="atolye-datv-repeater"
+BR2_TARGET_GENERIC_ISSUE="Welcome to yATOLYE DATV Repeater"
 BR2_ROOTFS_DEVICE_CREATION_DYNAMIC_MDEV=y
 BR2_TARGET_GENERIC_ROOT_PASSWD="analog"
 BR2_TARGET_GENERIC_GETTY_PORT="ttyPS0"
@@ -62,3 +62,5 @@ BR2_TARGET_ROOTFS_CPIO_UIMAGE=y
 BR2_PACKAGE_HOST_DOSFSTOOLS=y
 BR2_PACKAGE_HOST_GENIMAGE=y
 BR2_PACKAGE_HOST_MTOOLS=y
+BR2_PACKAGE_FFMPEG=y
+BR2_PACKAGE_LAME=y
-- 
2.43.0

